src/App.js

```diff
@@ -1,21 +1,55 @@
-import 'antd/dist/antd.css';
-import React from 'react';
+import "antd/dist/antd.css";
+import React, {Component} from "react";
 import ErrorBoundary from "./components/error/ErrorBoundary";
-import './App.css';
+import "./App.css";
 import Header from "./components/header/Header";
 import Home from "./components/home/Home";
 import Footer from "./components/footer/Footer";
+import {MOVIES} from "./constants/Constant";
 
-function App() {
-  return (
-    <ErrorBoundary>
-      <div className='app'>
-        <Header/>
-        <Home/>
-        <Footer/>
-      </div>
-    </ErrorBoundary>
-  )
+class App extends Component {
+  state = {
+    movies: MOVIES
```

Comment:
В этом компоненте state с фильмами надо было сетать в методе componentDidMount, я про это говорил на последнем Q&A
```suggestion
constructor() {
    super(props);
    state: {
        movies: [];
    }
}

    componentDidMount () {
        this.setState({ movies: MOVIES })
    }
```

Reply:
Done